user_docs/en/userGuide.t2t

```diff
@@ -2785,10 +2785,18 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 
 ++ Log Viewer ++[LogViewer]
 The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you last started NVDA.
-Using NVDA+F1 will open the log viewer and display developer information about the current navigator object.
 
 Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it shows the most recent output since the Log viewer was opened.
 These actions are available under the viewer's Log menu.
+The file which is opened when you press NVDA+f1 is saved on your computer in the temporary folder "%temp%" of your Windows and is called NVDA.log.
+A log file is also saved in case NVDA crashes. The file is called NVDA_old.log and is saved in the same location.

```

Comment:
That's not true.
nvda-old.log is the log of the previous NVDA session (run). If it has crashed or not does not change anything, i.e. nvda-old.log is just nvda.log which is renamed when a new NVDA session is started.